### ISSUE TITLE
Error on "Add DbContext  Model Diagram"

### ISSUE THREAD

**Issue by mehmetilker** [ErikEJ/SqlCeToolbox#584]
### Steps to reproduce
After successful "Reverse Engineer", I clicked "Add DbContext  Model Diagram" and I have got following exception at the output.

TargetFramework: netcoreapp2.0

### Further technical details

```
System.ArgumentException: Error:
System.ArgumentException: No EF Core DbContext types found in the project
   at ReverseEngineer20.EfCoreModelBuilder.GetDbContextTypes(String outputPath, DbContextOperations operations) in C:\Code\SqlCeToolbox\src\GUI\ReverseEngineer20\ModelBuilder\EFCoreModelBuilder.cs:line 78
   at ReverseEngineer20.EfCoreModelBuilder.GenerateDebugView(String outputPath) in C:\Code\SqlCeToolbox\src\GUI\ReverseEngineer20\ModelBuilder\EFCoreModelBuilder.cs:line 22
   at ReverseEngineer20.Program.Main(String[] args) in C:\Code\SqlCeToolbox\src\GUI\efpt\Program.cs:line 25

Parameter name: processResult
   at EFCorePowerTools.Handlers.ModelAnalyzerHandler.Generate(String outputPath, Project project, GenerationType generationType) in C:\projects\sqlcetoolbox\src\GUI\EFCorePowerTools\Handlers\ModelAnalyzerHandler.cs:line 52

```

Toolbox/Power Tools version: (found in About dialog - blue questionmark icon)
Version 4.7.458.0

SQL Server Compact 4.0 in GAC - No
SQL Server Compact 4.0 DbProvider - No

SQL Server Compact 4.0 DDEX provider - No
SQL Server Compact 4.0 Simple DDEX provider - Yes
SQLite ADO.NET Provider included: 1.0.105.1
SQLite EF6 DbProvider in GAC - No

System.Data.SQLite DDEX provider - No
SQLite Simple DDEX provider - Yes

Database engine: Sql Server 2017

Visual Studio or SSMS version: VS 2017.4.5

**Comment by ErikEJ** [comment]
Can you share a repro project?

**Comment by mehmetilker** [comment]
[ConsoleAppEFCoreToolingTest.zip](https://github.com/ErikEJ/SqlCeToolbox/files/1527879/ConsoleAppEFCoreToolingTest.zip)

Steps I have followed:
-I created a console app.
-Power Tools> Reverse Engineer
--Created connection (was ready already)
--Chose one table, save.
-Build the project with newly generated files.

Then "Power Tools" > Add Db Context model diagram.
No errors shown but I can see exception log on output window.

I have attached the example solution.
